src/main/resources/data-mapping.json

```diff
@@ -718,6 +718,624 @@
           }
         }
       ]
+    },
+    {
+      "orderType": "ListedPrintSerial",
+      "mappings": [
+        {
+          "field": "TITLE",
+          "dataSource": {
+            "from": "//datafield[@tag='245']/*",
+            "combinator": "concat"
+          }
+        },
+        {
+          "field": "PUBLISHER",
+          "dataSource": {
+            "from": "//datafield[@tag='260']/subfield[@code='b']"
+          }
+        },
+        {
+          "field": "FUND_CODE",
+          "dataSource": {
+            "from": "//FundCode",
+            "translation": "lookupFundId"
+          }
+        },
+        {
+          "field": "INSTRUCTIONS",
+          "dataSource": {
+            "from": "//OrderNotes"
+          }
+        },
+        {
+          "field": "QUANTITY",
+          "dataSource": {
+            "from": "//Quantity",
+            "default": "1",
+            "translation": "toInteger"
+          }
+        },
+        {
+          "field": "VENDOR_REF_NO",
+          "dataSource": {
+            "from": "//YBPOrderKey"
+          }
+        },
+        {
+          "field": "CREATED_DATE",
+          "dataSource": {
+            "from": "//OrderPlaced",
+            "translation": "toDate"
+          }
+        },
+        {
+          "field": "OWNER",
+          "dataSource": {
+            "from": "//BaseAccount",
+            "default": "Main"
+          }
+        },
+        {
+          "field": "APPROVAL_STATUS",
+          "dataSource": {
+            "default": "APPROVED",
+            "translation": "lookupWorkflowStatusId",
+            "translateDefault": true
+          }
+        },
+        {
+          "field": "VENDOR",
+          "dataSource": {
+            "default": "GOBI Library Solutions"
+          }
+        },
+        {
+          "field": "FUND_PERCENTAGE",
+          "dataSource": {
+            "default": "100"
+          }
+        },
+        {
+          "field": "PRODUCT_ID_TYPE",
+          "dataSource": {
+            "default": "ISBN"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_FROM_DATE",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_INTERVAL",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_TO_DATE",
+          "dataSource": {
+            "default": "NA"
```

Comment:
if this is a date-time field in the schema I don't think "NA" is going to fly.  Probably better to just leave undefined.